mkdocs/docs/reference/tasks/metrics-collect.md

```diff
@@ -32,17 +23,21 @@ start:
 ## Inputs
 | Field name | Field type | Description | Required |
 | ----- | ---- | ----------- | -------- |
-| time | string | Duration of the `metrics/collect` task run. Specified in the [Go duration string format](https://golang.org/pkg/time/#ParseDuration). Default value is `5s`. | No |
-| payloadURL | string | URL of JSON-encoded data. If this field is specified, the metrics collector will send HTTP POST requests to versions, and the POST requests will contain this JSON data as payload. | No |
-| versions | [][Version](#version) | A non-empty list of versions. | Yes |
+| numQueries | int | number of requests sent to each version. Default value is 100. | No |
```

Comment:
Is this number of requests *to be* sent to each version?

Reply:
Since this has precedence over time, perhaps that should be stated explicitly.

Reply:
Fixed `to be sent`. Also explicitly specified number has precedence over time in the time field.